site/docs/encoding.md

```diff
@@ -61,64 +88,88 @@ The following table lists supported channels for each mark type.
+: Nominal data, also known as categorical data, differentiates between values based only on their names or categories.  For example, gender, nationality, music genre, names are all nominal data.  Numbers maybe used to represent the variables but the number do not determines magnitude or ordering.  For example, if a nominal variable contains three values 1, 2, and 3.  We cannot claim that 1 is less than 2 nor 3.  
+
+Note that data `type` here describes semantic of the data rather than primitive data types in programming language sense (`number`, `string`, etc.).  The same primitive data type can have different type of measurement.  For example, numeric data can represent quantitative, ordinal, and nominal data.  
 
-Note that data types describe the more than just the raw data type (e.g. `number`, `string`, `date`) as for example a field with only numbers can be quantitative but also ordinal when the numbers encode a category.
+| Property      | Type          | Description    |
+| :------------ |:-------------:| :------------- |
+| type          | String        | The encoded field's [type of measurement ](https://en.wikipedia.org/wiki/Level_of_measurement).  This can be either a full type name (`"quantitative"`, `"temporal"`, `"ordinal"`,  and `"nominal"`) or an initial character of the type name (`"Q"`, `"T"`, `"O"`, `"N"`).  This property is case insensitive. |
 
 {:#inline}
-### Inline Transforms
+### Field Transforms
 
-To facilitate data exploration, Vega-Lite provides the following inline transforms as a part of the field definition: binning (`bin`), time unit conversion (`timeUnit`), aggregation (`aggregate`), sort (`sort`).  
+To facilitate data exploration, Vega-Lite provides inline field transforms as a part of the channel definition. If a `field` is provided, the channel definition supports the following transformations:
 
-After the specified top-level `transform`s are executed, inline transforms are executed in this order: binning, time unit conversion, aggregation, and sorting.
+<!--TODO: revise description in this table-->
 
 | Property      | Type          | Description    |
 | :------------ |:-------------:| :------------- |
-| [bin](bin.html) | Boolean &#124; Object        | Boolean flag / configuration object for binning.   |
-| [timeUnit](timeunit.html)| String        | Property for converting time unit.            |
-| [aggregate](aggregate.html) | String        | Aggregation function for the field (e.g., `mean`, `sum`, `median`, `min`, `max`, `count`).  |
-| [sort](sort.html) | String &#124; Object        | Sort order for a particular field.  This can be string (`"ascending"`, `"descending"`, or `"unsorted"`) or a sort field definition object for sorting by an aggregate calculation of a specified sort field.  If unspecified, the default value is `ascending`. |
+| [bin](bin.html)<sup>1</sup> | Boolean &#124; Object | Boolean flag / configuration object for binning.    |
```

Comment:
I know but that will put footnote at the end of the page, which is obviously too far away in this case.